jberet-se/src/main/java/org/jberet/se/BatchSEEnvironment.java

```diff
@@ -72,12 +80,27 @@ public final class BatchSEEnvironment implements BatchEnvironment {
     static final String THREAD_POOL_REJECTION_POLICY = "thread-pool-rejection-policy";
     static final String THREAD_FACTORY = "thread-factory";
 
+    static final String DB_USER_KEY = "db-user";
+    static final String DB_PASSWORD_KEY = "db-password";
+
     public BatchSEEnvironment() {
         configProperties = new Properties();
         final InputStream configStream = getClassLoader().getResourceAsStream(CONFIG_FILE_NAME);
         if (configStream != null) {
             try {
                 configProperties.load(configStream);
+                if (configProperties.getProperty(JOB_REPOSITORY_TYPE_KEY).equals(REPOSITORY_TYPE_JDBC)) {
```

Comment:
here we are doing this only for jdbc repository, which may be too restrictive. MongoDB repository also uses db-user and db-password, so it will be good to cover that as well. Probably make it as loose as, if they are present, they will be parsed.